<launch>
    <node name="drag_centerline_planner" pkg="rr_iarrc" type="drag_centerline_planner" output="screen">
        <!--PID Controller Constants-->
        <param name="PID_kP" type="double" value="0.006" /> <!-- 0.003 -->
        <param name="PID_kI" type="double" value="0.0" />
        <param name="PID_kD" type="double" value="0.000" />

        <param name="speed" type="double" value="2.1" /> <!-- 3.1-->

        <param name="maxTurnLimitRadians" type="double" value="0.44" />

        <param name="subscription_node" type="string" value="/camera_center/lines/detection_img_transformed"/>
    </node>
</launch>
